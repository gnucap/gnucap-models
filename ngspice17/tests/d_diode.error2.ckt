
d1 1 0 foo bar=1
.model foo sw
.op
.end
