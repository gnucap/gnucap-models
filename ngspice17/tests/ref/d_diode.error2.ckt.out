 
d1 1 0 foo bar=1 
           ^ ? d1: bad parameter bar ignored
d1: model foo is not a ngspice_dio
